### DISCUSSION TITLE
[Help needed] Export generated ssh key

### DISCUSSION THREAD

**Discussion by krendelhoff2** [zeapo/Android-Password-Store#1999] [Q&A] [answered]
Is there any way to get generated ssh private key?

**Comment by msfjarvis** [comment, marked as answer]
There is not. The point of generating a key on-device is keeping it as secure as the underlying platform can make it. Allowing secret keys to be exported would also encourage key re-use, which is a bad security practice that we'd prefer to not perpetuate.

**Reply by krendelhoff2** [reply]
The situation is I lost access to my github account and the only trusted ssh key left is locked inside password store. I need it to prove my identity. Is it possible to get it somehow?

**Reply by msfjarvis** [reply]
Sorry to hear about your predicament. Unfortunately the key cannot be extracted from the app.

**Reply by krendelhoff2** [reply]
Nevertheless, thank you very much.
